src/plugins/geoservices/osm/qgeotilefetcherosm.cpp

```diff
@@ -59,17 +59,17 @@ QGeoTiledMapReply *QGeoTileFetcherOsm::getTileImage(const QGeoTileSpec &spec)
     switch (spec.mapId()) {
         case 1:
             // opensteetmap.org street map
-            request.setUrl(QUrl(QStringLiteral("http://otile1.mqcdn.com/tiles/1.0.0/map/") +
+            request.setUrl(QUrl(QStringLiteral("https://tile.openstreetmap.org/") +
                                 QString::number(spec.zoom()) + QLatin1Char('/') +
                                 QString::number(spec.x()) + QLatin1Char('/') +
                                 QString::number(spec.y()) + QStringLiteral(".png")));
             break;
         case 2:
             // opensteetmap.org satellite map
-            request.setUrl(QUrl(QStringLiteral("http://otile1.mqcdn.com/tiles/1.0.0/sat/") +
+            request.setUrl(QUrl(QStringLiteral("https://basemap.nationalmap.gov/arcgis/rest/services/USGSImageryOnly/MapServer/tile/") +
```

Comment:
This one however doesn't quite work - the tiles are in wrong order. To me it seems the x and y are reversed.

Reply:
Ah, sigh. There is no point in using this basemap. It's so restricted (zoom 8 max) as to be useless. All the other maps we might use we can't use legally. Even personal use is heavily restricted.  I'm looking at https://scihub.copernicus.eu/dhus/#/home as an alternative since they also have open access.

Reply:
Although not documented at the old site: 
```https://ibasemaps-api.arcgis.com/arcgis/rest/services/Ocean/World_Ocean_Base/MapServer/tile/{z}/{y}/{x}?token=<ACCESS_TOKEN> ``` suggests they've moved all the apis ... in the wrong direction :)